src/css/jwplayer/imports/reset.less

```diff
@@ -17,4 +17,8 @@
     font-stretch: inherit;
     /* non-standard prefixed style used in Chrome, Safari, and some MS browsers.  Not Autoprefixed */
     -webkit-tap-highlight-color: rgba(255, 255, 255, 0);
-}
\ No newline at end of file
+    
+    &:focus {
+        outline: none;
```

Comment:
I meant to ask you about that. The better solution here is to add `jw-button-color` in sharing so both focus and hover states are fixed along with the skin color.